Modules/Scripted/ExtensionWizard/ExtensionWizardLib/EditExtensionMetadataDialog.py

```diff
@@ -56,9 +57,9 @@ def __init__(self, parent):
 #
 # =============================================================================
 class EditExtensionMetadataDialog:
-    project = _map_property(lambda self: self.ui.nameEdit, "text")
-    category = _map_property(lambda self: self.ui.categoryEdit, "text")
-    description = _map_property(lambda self: self.ui.descriptionEdit, "plainText")
+    project = _map_property(lambda self: self.ui.nameEdit, _("text"))
```

Comment:
"text" must not be translated in these lines (this is a Qt object property name)